doc/requirements.txt

```diff
@@ -1,22 +1,8 @@
 -r ../requirements.txt
-# Enforce latest version on ReadTheDocs
+
 sphinx>=4.2.0
```

Comment:
Upgrade to Sphinx 4.3.0

Reply:
Forcing the version to the very latest might force Sphinx to be reinstalled in each run, instead of using the one in the container from BTD. In this case, that is not a problem, because it is using v4.3.0 already. Nonetheless, we should be cautious.

Reply:
See #15.

Reply:
merged.